src/Gelf/Logger.php

```diff
@@ -60,7 +60,11 @@ public function __construct(
      */
     public function log($level, $rawMessage, array $context = array())
     {
-        $message = $this->initMessage($level, $rawMessage, $context);
+        if (!($rawMessage instanceof MessageInterface)) {
+            $message = $this->initMessage($level, $rawMessage, $context);
+        } else {
+            $message = $rawMessage;
+        }
```

Comment:
The else is unneccessary. Just rename $rawMessage to $message and drop the else-clause.